crates/api_common/src/request.rs

```diff
@@ -53,7 +53,9 @@ pub async fn fetch_link_metadata(
   // https://github.com/LemmyNet/lemmy/issues/1964
   let html_bytes = response.bytes().await.map_err(LemmyError::from)?.to_vec();
 
-  let opengraph_data = extract_opengraph_data(&html_bytes, url).unwrap_or_default();
+  let opengraph_data = extract_opengraph_data(&html_bytes, url)
+    .map_err(|e| warn!("{e}"))
```

Comment:
Added logging here.

Reply:
I would put this under info or even verbose. No reason to spam server logs with warnings about this.

Reply:
K, will use verbose.